packages/fast-components-react-base/src/dialog/index.ts

```diff
@@ -0,0 +1,4 @@
+import Dialog from "./dialog";
+
+export default Dialog;
+export * from "./dialog";
```

Comment:
I believe this needs to end in a newline

Reply:
Very odd because, there is a newline on my machine.